src/oauth2.h

```diff
@@ -26,29 +26,32 @@
 
 namespace google {
 
+constexpr const char kDefaultTokenEndpoint[] =
+  "https://www.googleapis.com/oauth2/v3/token";
+
 class OAuth2 {
  public:
-  OAuth2(const Environment& environment) : environment_(environment) {}
+  OAuth2(const Environment& environment)
+    : environment_(environment), token_endpoint_(kDefaultTokenEndpoint) {}
 
   std::string GetAuthHeaderValue();
 
-  template<class Request>
```

Comment:
Why did you remove this? If it's unused, you need to mention that in the PR description.

Reply:
Yeah it's unused. I'll update the description.